remove pytest pretty from run-unit-tests (#3007)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Removed `Flake8` from **pre-commit**, as `ruff` covers its basic rules.
* Improved log readability by silencing non-critical `neo4j` (content graph infrastructure) logs.

> * Fixed an issue with **run-unit-tests** command on python2 content items.

Reviewer: ```suggestion
* Fixed an issue where **run-unit-tests** failed on python2 content items.
```
